codice/ddf
# DDF-4509 Unable to view workspace details when local catalog is disabled

#### What does this PR do?
When local catalog is disabled you are unable to view workspace details.

Viewing details sends a /cql request to every source with federation set to enterprise. None of those (unless local catalog is enabled) is able to return the workspace metacard and results in a page of no results found when it should be showing workspace properties.

This allows the metacard component to send to the `local` UI node in the case you are viewing a workspace's details.
#### Who is reviewing it? 
@bellcc 
@bdeining 
@willwill96 
@GabrielFabian 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@millerw8

#### How should this be tested?

- Ingest data into DDF
- Run query for that data and save a workspace
- Enable local catalog
- View workspace details
- See workspace details under table or inspector
- Turn local catalog off
- View workspace details again
- See that you retrieve workspace details again rather than "no results found"

#### Any background context you want to provide?
#### What are the relevant tickets?
Fixes: #4509 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
